data/json/items/tool/deployable.json

```diff
@@ -367,5 +367,23 @@
     "symbol": "7",
     "color": "yellow_red",
     "use_action": { "type": "deploy_furn", "furn_type": "f_manual_tire_changer" }
+  },
+  {
+    "id": "folded_wheelchair",
+    "type": "TOOL",
+    "name": { "str": "folded Foldable wheelchair" },
+    "description": "A folded Foldable wheelchair.",
+    "weight": "10586 g",
+    "volume": "13500 ml",
+    "longest_side": "0 cm",
```

Comment:
I guess this should be set to something?

Reply:
Interesting - I thought I had done this, I even specifically remember looking up the average size of a wheelchair. Guess I messed it up somehow, thanks for the catch. I set it to 92cm (average height is 36 inches which converts to 91.44 cm).